src/TableColumn/ExternalLink.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace atk4\ui\TableColumn;
+
+use atk4\data\Field;
+
+/**
+ * Implements Column helper for grid.
+ * Use
+ *   new Link('http://yahoo.com?id={$id}');
+ * or
+ *   new Link(['order', 'id'=>'id' ]);
+ * or
+ *   new Link(['order', 'id' ]);
+ * or
+ *   new Link([['order', 'x'=>$myval], 'id' ]);.
+ */
+class ExternalLink extends Generic
```

Comment:
How is this class better than `TableColumn\Link` class we already have?

Reply:
i can merged tags in link, it was one of the first columns i have done